multiqc/modules/samtools/samtools.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from samtools stats command"""
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re
+
+from multiqc import config, BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='samtools',
+        anchor='samtools', target='samtools',
+        href='http://www.htslib.org',
+        info="Samtools is a suite of programs for interacting"\
+        " with high-throughput sequencing data.")
+        self.samtools_data = dict()
+        for f in self.find_log_files(config.sp['samtools']):
+            s_name = f['s_name'].split(".txt")[0]
```

Comment:
As above, might be nicer to just add `.txt` to `fn_clean_exts` in the config (I'm surprised it's not there already!)